sdcm/utils/syslogng.py

```diff
@@ -37,6 +37,12 @@ def syslogng_confpath(self) -> str:
     def syslogng_port(self) -> Optional[int]:
         return ContainerManager.get_container_port(self, "syslogng", SYSLOGNG_PORT)
 
+    @property
+    def syslogng_log_dir(self) -> Optional[str]:
+        # This should work after process that had run container was terminated and another one starts
+        # that is why it is not using variable to store log directory instead
```

Comment:
This what also should be in the commit message. From the commit message point of view the bug that gets fixed here is unclear.

Reply:
It is, I've added couple of words to make it easy to grasp